VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -50,7 +50,7 @@ public class Subscription: VIMModelObject
     /// Represents the Subscription object as a Dictionary
     public var toDictionary: [AnyHashable: Any]
     {
-        let dictionary = ["comment": self.comment ?? false,
+        let dictionary: [String: Any] = ["comment": self.comment ?? false,
```

Comment:
@TomCarey Thank you for catching this.  Please change the type to [AnyHashable: Any] to match the property signature and left align the entries in the dictionary.